ssz/src/main/java/net/consensys/cava/ssz/SSZWriter.java

```diff
@@ -428,4 +429,13 @@ default void writeAddressList(Bytes... elements) {
   default void writeBooleanList(boolean... elements) {
     SSZ.encodeBooleanListTo(elements, this::writeSSZ);
   }
+
+  /**
+   * Write a java.util.List of bytes.
+   *
+   * @param elements The bytes to write as a java.util.List.
+   */
+  default void writeBytesList(List<Bytes> elements) {
```

Comment:
Move up so it's immediately below `default void writeBytesList(Bytes... elements)`?

Reply:
The format the class seemed to follow was 
  - writes single types - i.e.:
    - writeHash
    - writeString
  - write multiple varags types - i.e.:
    - writeHashList
    - writeStringList

so it made sense to me to add the write java.util.List methods below the entire multiple types block. Your call where it should belong.

Reply:
The way I see it, `writeBytesList(List<Bytes> elements)` is the same as `writeBytesList(Bytes... elements)`, differing only in the container part of the parameter type (one is an array/vararg, the other a list, both containing zero or more `Bytes` instances). So having them appear next to each other in the code _and the javadoc_ would be nice.

Reply:
Moved